packages/snap/snap.config.ts

```diff
@@ -14,14 +14,14 @@ const config: SnapConfig = {
         };
         // Update the definitions
         definePlugin.definitions['process.env.DEBUG'] = 'process.env.DEBUG';
-        definePlugin.definitions['process.env.DEBUG)'] = '"false")';
-        definePlugin.definitions['process.env.DEBUG;'] = '"false";';
+        /* definePlugin.definitions['process.env.DEBUG)'] = '"false")';
+        definePlugin.definitions['process.env.DEBUG;'] = '"false";'; */
       }
     }
-    // if (res.optimization) {
```

Comment:
Can you please turn on the optimization again. It makes the snap bundle much smaller.

Reply:
optimization turned on